index.js

```diff
@@ -485,11 +485,23 @@ module.exports = function Bus() {
                     delete $meta.callback;
                     return cb.apply(this, applyArgs);
                 } else if (!fn) {
-                    var type;
-                    // noinspection JSUnusedAssignment
-                    (methodName && (type = bus.local) && (fn = type[methodName]) && (local = true)) ||
-                    (methodName && (!bus.socket) && (fn = findMethod(mapLocal, mapLocal, methodName, methodType)) && !(local = false)) ||
-                    ((type = bus[typeName]) && (fn = type['master.' + methodType]) && (local = false));
+                    if (methodName) {
+                        fn = bus.local[methodName];
```

Comment:
There was such an unconditional check just after the next if: https://github.com/softwaregroup-bg/ut-bus/blob/fix/cleanup/index.js#L556 
It should all be ok as local is an empty object by default: https://github.com/softwaregroup-bg/ut-bus/blob/master/index.js#L269